### PR TITLE
Reduce dependences in DGEtools

### PR DESCRIPTION
**Pull request by klmedeiros-ag**
Includes work that reduces dependences on DGEtools by 6 packages. 

(There's also a lot of commits on this PR that are only here because of the work I did to fix my own develop branch. cb4ds/develop already has that work, and my apologies that it lingers here.)

### CODE REVIEW

File: R/rsqCalc.R [hunk 1]
```diff
@@ -26,8 +25,8 @@ rsqCalc <- function(normMatrix, fit) {
                             msg = "fit must be of class 'MArrayLM'.")
     assertthat::assert_that(is.numeric(as.matrix(normMatrix)),
                             msg = "All of the entries in normMatrix must be numeric.")
-
-    sst <- Matrix::rowSums(normMatrix^2)
+    browser()
```
Comment (cb4ds) [inline R/rsqCalc.R]: remove please, we shouldn't have browser() in a function.

File: R/rsqCalc.R [hunk 2]
```diff
@@ -26,8 +25,8 @@ rsqCalc <- function(normMatrix, fit) {
...
+    sst <- rowSums(normMatrix^2)
```
Comment (cb4ds) [inline R/rsqCalc.R:29]: where is rowSums coming from if not Matrix?  I think the base version is "rowsum"

**Review by cb4ds — CHANGES_REQUESTED** [review]
Minor change needed + one question to answer.

File: R/rsqCalc.R (hunk 2 above)
Comment (klmedeiros-ag) [reply R/rsqCalc.R:29]: @cb4ds I found a base `rowSums()` function in addition to the `rowsum()` function: image

Comment (cb4ds) [reply R/rsqCalc.R:29]: excellent, thanks!

**Review by cb4ds — DISMISSED** [review]

**Review by cb4ds — CHANGES_REQUESTED** [review]
https://travis-ci.org/github/cb4ds/DGE.Tools2/builds/748643382
Tests still failing

**Review by cb4ds — APPROVED** [review]

**Pull request merged**